ngmix/medsreaders.py

```diff
@@ -134,8 +135,12 @@ def get_obslist(self, iobj, weight_type='weight'):
         """
         obslist = ObsList()
         for icut in range(self._cat['ncutout'][iobj]):
-            obs = self.get_obs(iobj, icut, weight_type=weight_type)
-            obslist.append(obs)
+            try:
+                obs = self.get_obs(iobj, icut, weight_type=weight_type)
+                obslist.append(obs)
+            except GMixFatalError as err:
+                print('zero weight observation found, skipping')
```

Comment:
We have been punting on logging in ngmix for a while. Maybe start now?

Reply:
definitely soon, but I need to go into production on a short timescale so need to punt again